File: PHP/PartLaptop.php
<?php
require_once 'Barang.php';

class PartLaptop extends Barang {
    private string $noPart;
    private string $manufaktur;
    private string $kondisi;

    public function __construct(
        int $id = 0, string $nama = "", int $harga = 0, int $stok = 0, string $gambar = "",
        string $noPart = "", string $manufaktur = "", string $kondisi = ""
    ) {
        parent::__construct($id, $nama, $harga, $stok, $gambar);
        $this->noPart = $noPart;
        $this->manufaktur = $manufaktur;
        $this->kondisi = $kondisi;
    }

    public function getNoPart(): string { return $this->noPart; }
    public function setNoPart(string $noPart) { $this->noPart = $noPart; }

    public function getManufaktur(): string { return $this->manufaktur; }
    public function setManufaktur(string $manufaktur) { $this->manufaktur = $manufaktur; }

    public function getKondisi(): string { return $this->kondisi; }
    public function setKondisi(string $kondisi) { $this->kondisi = $kondisi; }
}

File: Folder Python/PartLaptop.py
from Barang import Barang

class PartLaptop(Barang):
    def __init__(self,
                 id=0, nama="", harga=0, stok=0,
                 no_part="", manufaktur="", kondisi=""):
        super().__init__(id, nama, harga, stok)
        self._no_part = no_part
        self._manufaktur = manufaktur
        self._kondisi = kondisi

    @property
    def no_part(self): return self._no_part
    @no_part.setter
    def no_part(self, v): self._no_part = v

    @property
    def manufaktur(self): return self._manufaktur
    @manufaktur.setter
    def manufaktur(self, v): self._manufaktur = v

    @property
    def kondisi(self): return self._kondisi
    @kondisi.setter
    def kondisi(self, v): self._kondisi = v
